src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/TournamentQueries.cs

```diff
@@ -4,25 +4,27 @@
     using Data.Queries.Common;
     using Data.Queries.Tournament;
     using Domain.TournamentsAggregate;
-    using Entities;
-    using System;
     using System.Collections.Generic;
     using System.Linq;
+    using Data.Queries.Division;
+    using Data.Queries.Group;
     using System.Linq.Expressions;
-    using VolleyManagement.Data.Queries.Division;
-    using VolleyManagement.Data.Queries.Group;
+    using System;
+    using Entities;
 
+#pragma warning disable S1200 // Classes should not be coupled to too many other classes (Single Responsibility Principle)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)